### DISCUSSION TITLE
OMI glTF Working Group Meeting 2024-03-14

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/gltf-extensions#215] [General]
This meeting is on 2024-03-14 at 21:30 UTC / 2:30 PM PST in the [OMI Discord](https://discord.gg/J4vyAWFgkj) within the "Weekly Meeting" voice channel. During the meeting, we will be using the #omi-gltf-extensions text channel to post links and engage in further discussion.

To be notified of this meeting and others, subscribe to the [OMI Meetings and Events Calendar](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) or add yourself to the @omi-gltf-subgroup role in the #roles channel of the OMI Discord.

**Comment by antpb** [comment]
noting that there werent any high level topics to discuss today. Open floor discussions about gltf and we'll document as we move along :)

**Comment by andybak** [comment]
> add yourself to the @omi-gltf-subgroup role in the #roles channel of the OMI Discord.

I'm not seeing a roles channel?
